sdk/nextjs/src/server/initialize.ts

```diff
@@ -72,10 +72,10 @@ export const initialize = async (
     }
 
     if (!initializeAlreadyCalled) {
-        client.synchronizeBootstrapData(config, user, getUserAgent(options))
+        client.synchronizeBootstrapData(config, user, await getUserAgent(options))
```

Comment:
is this not part of the same function as where `userAgent` is being called? seems like another await for no reason..